pptx/oxml/text.py

```diff
@@ -324,8 +326,10 @@ class CT_TextField(BaseOxmlElement):
     <a:fld> field element, for either a slide number or date field
     """
 
+    id = RequiredAttribute("id", XsdString)
     rPr = ZeroOrOne("a:rPr", successors=("a:pPr", "a:t"))
     t = ZeroOrOne("a:t", successors=())
+    type = OptionalAttribute("type", ST_FieldType)
```

Comment:
type is a python built-in function. Best to use a different name.

Reply:
See comment above re: id. There is a precedent for this as well in the CT_Placeholder class in [pptx/oxml/shapes/shared.py](https://github.com/scanny/python-pptx/blob/d043334b984736a7a2ade3fb6f9adcdd97b3e8f5/pptx/oxml/shapes/shared.py#L315)